installer_scripts/install-yb-voyager

```diff
@@ -314,11 +315,26 @@ update_yb_voyager_bashrc() {
 
 
 install_yb_voyager() {
+	os=$1
 	output "Installing yb-voyager:${VERSION}."
 	if [ "${VERSION}" == "latest" ]
 	then
-		$GO install github.com/yugabyte/yb-voyager/yb-voyager@${YB_VOYAGER_GIT_HASH}
-		sudo mv -f $HOME/go/bin/yb-voyager /usr/local/bin
+		if [ "$os" == "linux" ]
+		then
+			binary_name="voyager-linux-amd64"
+		elif [ "$os" == "darwin" ]
+		then
+			arch="$(uname -m)"
+			if [ "${arch}" == "arm64" ]
+			then
+				binary_name="voyager-darwin-arm64"
+			else
+				binary_name="voyager-darwin-amd64"
+			fi
+		fi
+
+		wget https://github.com/yugabyte/yb-voyager/releases/download/yb-voyager%2Fv${VOYAGER_RELEASE_TAG}/${binary_name} 1>&2
```

Comment:
A voyager "release" must be a TAR file. And we should be able to bundle as many files in it as we want (e.g. base_ora2pg.conf).